### PR TITLE
EG-485 update endpoint

### PR DESCRIPTION
**Pull request by LesWilson**
Updated PR to fix the issue with unverfied commits after deletion of GPG key

### CODE REVIEW

File: app/repositories/ApprovalRepository.scala [hunk 1]
```diff
@@ -110,4 +111,29 @@ class ApprovalRepositoryImpl @Inject() (implicit mongoComponent: ReactiveMongoCo
     //$COVERAGE-ON$
   }
 
+  def changeStatus(id: String, statusInfo: ApprovalProcessStatusChange): Future[RequestOutcome[Boolean]] = {
```
Comment (cmrhmrc) [inline app/repositories/ApprovalRepository.scala:114]: Think the Boolean should be Unit, it never returns false, either succeeds or fails

File: app/services/ReviewService.scala [hunk 2]
```diff
@@ -50,4 +52,12 @@ class ReviewService @Inject() () {
     Future(Right(mockData()))
   }
 
+  def changeStatus(id: String, info: ApprovalProcessStatusChange): Future[RequestOutcome[Boolean]] = {
```
Comment (cmrhmrc) [inline app/services/ReviewService.scala:55]: as repository return

File: app/repositories/ApprovalRepository.scala (hunk 1 above)
Comment (LesWilson) [reply app/repositories/ApprovalRepository.scala]: updated and pushed

File: app/services/ReviewService.scala (hunk 2 above)
Comment (LesWilson) [reply app/services/ReviewService.scala]: updated and pushed

File: test/controllers/ProcessReviewControllerSpec.scala [hunk 3]
```diff
@@ -179,4 +180,107 @@ class ProcessReviewControllerSpec extends WordSpec with Matchers with ScalaFutur
...
+      }
+
+    }
+
+    "the request is invalid" should {
+
+      trait InvalidTest extends Test {
+        MockReviewService
+          .changeStatus(validId, statusChangeInfo)
+          .returns(Future.successful(Right(true)))
```
Comment (cmrhmrc) [inline test/controllers/ProcessReviewControllerSpec.scala:212]: Right(())

File: test/controllers/ProcessReviewControllerSpec.scala [hunk 4]
```diff
@@ -179,4 +180,107 @@ class ProcessReviewControllerSpec extends WordSpec with Matchers with ScalaFutur
...
   }
 
+  "Calling the approvalReviewComplete action" when {
+
+    "the request is valid" should {
+
+      trait ValidTest extends Test {
+        MockReviewService
+          .changeStatus(validId, statusChangeInfo)
+          .returns(Future.successful(Right(true)))
```
Comment (cmrhmrc) [inline test/controllers/ProcessReviewControllerSpec.scala:190]: Right(())

**Pull request merged**